README.md

```diff
@@ -91,32 +109,39 @@ $ patch -p1 &lt; animation-interface.patch
 $ patch -p1 &lt; netanim-python.patch
 $ patch -p1 &lt; sta-wifi-scan.patch
 </pre>
-c. Configure ns-3 <br/>
+c. Fix the wscript error in the tap-bridge <br/>
```

Comment:
This patch is for the [issue] (https://www.nsnam.org/bugzilla/show_bug.cgi?id=1990) about using ns-3 with Ubuntu.